lsp-mode.el

```diff
@@ -7192,19 +7192,32 @@ nil."
 
 
 ;; Download URL handling
-(cl-defun lsp-download-install (callback error-callback &key url store-path &allow-other-keys)
-  (let ((url (lsp-resolve-value url))
-        (store-path (lsp-resolve-value store-path)))
+(cl-defun lsp-download-install (callback error-callback &key url store-path gzip? zip? &allow-other-keys)
```

Comment:
ZIP? and GZIP? are exclusive, no? In that case it might be better to have a `container?` parameter, which could be one of a few keywords like `:gzip`, `:zip`.